<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>200</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="actFont"/>
   <addaction name="actCascade"/>
   <addaction name="actTile"/>
   <addaction name="actCloseAll"/>
   <addaction name="actViewMode"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actFont">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="actCascade">
   <property name="text">
    <string>级联</string>
   </property>
  </action>
  <action name="actTile">
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
  <action name="actCloseAll">
   <property name="text">
    <string>关闭全部子窗口</string>
   </property>
  </action>
  <action name="actViewMode">
   <property name="text">
    <string>显示模式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
